src/compute/pre_compute_app.rs

```diff
@@ -398,35 +401,60 @@ mod tests {
     // region download_input_files
     #[test]
     fn download_input_files_success_with_single_file() {
+        let container = GenericImage::new("kennethreitz/httpbin", "latest")
+            .with_exposed_port(80.tcp())
+            .with_wait_for(WaitFor::message_on_stderr("Listening at"))
+            .with_network("bridge")
```

Comment:
Shouldn't this be the default behavior ? Do we really need to specify it ?

Reply:
Could the container instance be created with a method to remove duplications and improve maintainability ?